### PR TITLE
Remove posterior_probabilities parameter.

### PR DESCRIPTION
**Pull request by ypriverol**
This PR removes the posterior_probabilities parameter, only using the percolator. #448 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_lfq.config [hunk 1]
```diff
@@ -28,7 +28,6 @@ params {
     labelling_type = "label free sample"
     input = 'https://raw.githubusercontent.com/nf-core/test-datasets/quantms/testdata/lfq_ci/BSA/BSA_design_urls.tsv'
     database = 'https://raw.githubusercontent.com/nf-core/test-datasets/quantms/testdata/lfq_ci/BSA/18Protein_SoCe_Tr_detergents_trace_target_decoy.fasta'
-    posterior_probabilities = "fit_distributions"
```
Comment (jpfeuffer) [inline conf/test_lfq.config:31]: Haha I'm wondering if this will work with percolator

Comment (ypriverol) [reply conf/test_lfq.config:31]: Do you mean it was good to be removed? 😉

**Comment by daichengxin** [comment]
This module coud be removed when keep only percolator https://github.com/bigbio/quantms/tree/master/modules/local/openms/idpep

File: nextflow_schema.json [hunk 2]
```diff
@@ -521,13 +521,6 @@
                     "default": 0.15,
                     "fa_icon": "fas fa-filter"
                 },
-                "posterior_probabilities": {
-                    "type": "string",
-                    "description": "How to calculate posterior probabilities for PSMs:\n\n* 'percolator' = Re-score based on PSM-feature-based SVM and transform distance\n    to hyperplane for posteriors\n* 'fit_distributions' = Fit positive and negative distributions to scores\n    (similar to PeptideProphet)\n\n* 'mokapot' = Re-score based on PSM-feature-based semi-supervised learning algorithm introduced by Percolator",
-                    "fa_icon": "fas fa-list-ol",
-                    "default": "percolator",
-                    "enum": ["percolator", "fit_distributions", "mokapot"]
```
Comment (timosachsenberg) [inline nextflow_schema.json:529]: no mokapot in the future?

**Comment by ypriverol** [comment]
> This module could be removed when keeping only percolator https://github.com/bigbio/quantms/tree/master/modules/local/openms/idpep https://github.com/bigbio/quantms/tree/master/modules/local/openms/falsediscoveryrate

Should I remove it? @jpfeuffer @daichengxin? If you give me a 👍, I will in the same PR.

**Comment by jpfeuffer** [comment]
Not sure if FDR can be removed. It needs to be recalculated after ConsensusID

**Comment by ypriverol** [comment]
> Not sure if FDR can be removed. It needs to be recalculated after ConsensusID

Only IDPEP can be removed FDR is used all over the place.

**Comment by ypriverol** [comment]
@jpfeuffer @timosachsenberg @daichengxin Just remove also IDPEP as requested.

**Comment by ypriverol** [comment]
@jpfeuffer @timosachsenberg @daichengxin can you review this PR, and accept it 😉

**Review by daichengxin — APPROVED** [review]

**Review by timosachsenberg — APPROVED** [review]

**Pull request merged**